# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ooo/t2-distro.patch
# Copyright (C) 2004 - 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We really should send this upstream ASAP.

  - Rene Rebe <rene@exactcode.de>

--- ./patches/src680/apply.orig	2005-04-08 10:59:24.000000000 +0200
+++ ./patches/src680/apply	2005-04-18 09:49:25.000000000 +0200
@@ -73,6 +73,10 @@
 translate-org-za-win32: translate-org-za-common, translate-org-za-splash, Win32Common, Common
 # Novell
 NovellWin32: Win32, NovellOnly, NovellOnlyWin32, Lockdown, CalcSolver
+# www.t2-project.org
+T2Base : LinuxCommon, T2Only
+T2 : T2Base
+T264 : T2Base, 64bit
 
 # -------- [ Tag [ >= <tag> etc. ], ] patch sets --------
 
@@ -1057,6 +1061,11 @@
 testing-more-optimizations-ark.diff
 
 
+[ T2Only ]
+system-db-4.3-use-lpthread.diff
+xmlhelp-port-to-db-4.3.diff
+#buildfix-longnames-54730.diff
+
 [ RedhatOnly ]
 # Some palette loving
 redhat-standard-colors.diff, caolan
